signal/src/pcan_argc_fixed.h

```diff
@@ -0,0 +1,26 @@
+#ifndef SIGNAL_MICRO_KERNELS__SRC_PCAN_AGC_FIXED_H_
```

Comment:
There's an extra underscore in the include guards

Reply:
done